src/components/Table/Table.test.jsx

```diff
@@ -127,6 +127,58 @@ describe("Table", () => {
 			expect(page2Button).toHaveClass(selectedPageClass);
 			expect(page5Button).not.toBeVisible();
 		});
+
+		it("does not update `pageIndex` when all data is removed (refreshed)", async () => {
+			render(<EditableData />);
+
+			const refreshButton = screen.getByLabelText("Refresh");
+
+			expect(screen.getByText("1")).toHaveClass(selectedPageClass);
+
+			// refresh data, removing all data and then adding default data back in
+			await user.click(refreshButton);
```

Comment:
Better wrap user action with act() like below:
```suggestion
      import { act } from "@testing-library/react";

      await act(async () => {
        await user.click(refreshButton);
      });
```

Reply:
Why? I don't think this is necessary and am unsure how to confirm that this is needed or unneeded.

Reply:
If I run `yarn test Table` I don't see any warnings with the code as is. 
![Table test](https://github.com/avaya-dux/neo-react-library/assets/55896546/588aeefc-17d0-4fd0-81e1-ccac036019a4)

If I test Select (`yarn test Select`) I do see a warning for using `act`, so they will show up if needed as best I can tell. 
![Select test](https://github.com/avaya-dux/neo-react-library/assets/55896546/f120c1f7-6418-4134-9044-b2462f3bafd2)